nengo_viz/server.py

```diff
@@ -11,24 +11,38 @@
     from urllib.parse import unquote
 
 import nengo_viz.swi as swi
+import nengo_viz
 
 
 class Server(swi.SimpleWebInterface):
     """Web server interface to nengo_viz"""
 
     def swi_browse(self, dir):
-        self.script_path = '.'
         if self.user is None: return
         r = ['<ul class="jqueryFileTree" style="display: none;">']
         d = unquote(dir)
-        for f in sorted(os.listdir(os.path.join(self.script_path, d))):
-            ff = os.path.relpath(os.path.join(self.script_path, d,f), self.script_path)
-            if os.path.isdir(os.path.join(self.script_path, d, ff)):
-                r.append('<li class="directory collapsed"><a href="#" rel="%s/">%s</a></li>' % (ff,f))
+        ex_tag = '//examples//'
```

Comment:
> italics sounds good to me.

It's now in italics and says "built-in examples".  I tried bold and I tried centered, and those both looked weird to me, but I like this version with italics and with a non-standard directory name.